### ISSUE TITLE
Cleaning room order and carpets

### ISSUE THREAD

**Issue by krpn** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#185]
Please add cleaning room order (from Mi Home) and carpets visualization (for example, from Roborock S7) to the map.

**Comment by PiotrMachowski** [comment]
I don't think map data contain this info

**Comment by krpn** [comment]
It's available in MiHome:

![6DFD0AA2-E2F0-4A9C-9C63-B911AD51DAC6](https://user-images.githubusercontent.com/16793635/141004925-edd1402e-3c6d-4a78-a0c9-199f47cdb0f0.jpeg)

**Comment by krpn** [comment]
@PiotrMachowski, where I can get raw map file?

**Comment by PiotrMachowski** [comment]
Is room order the same thing as room ID?

https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#retrieving-map

**Comment by PiotrMachowski** [comment]
```
piotr.machowski.dev [at] gmail [dot] com
```

**Comment by krpn** [comment]
@PiotrMachowski, thank you. Sent.

**Comment by mikesrus** [comment]
@krpn Вы используете roborock S7? Есть ли возможность отображения имен комнат на карте и выбора комнаты для уборки? У меня карту показывает, но без ковров и названий комнат

**Comment by krpn** [comment]
@mikesrus, да, у меня Roborock S7. На карте комнаты и ковры не отображаются - об этом и тикет. Насчёт отправки в комнату не подскажу, так как не пользуюсь данным функционалом.
